ci/test.sh

```diff
@@ -3,6 +3,6 @@
 bundle
 
 for target in spec cucumber; do
-  bundle exec rake $target || true
```

Comment:
This had the advantage that it would always run both specs and cukes. Kind of bummer to lose that.

Reply:
Ah good point, I'm okay w/ rolling this back out as long as we can confirm that broken builds won't get marked green, that's my main concern.